### PR TITLE
Update readiness-assessment-fix.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated based on issue #6124

If there is an existing Autopilot Deployment profile (even if it is not assigned) the setting "Convert all targeted devices to Autopilot" must be set to "No"

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6124

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You can access Intune settings at the Microsoft Endpoint Manager [admin center](
 
 ### Autopilot deployment profile
 
-You shouldn't have any existing Autopilot profiles that target assigned or dynamic groups with Microsoft Managed Desktop devices. Microsoft Managed Desktop uses Autopilot to provision new devices.
+You shouldn't have any existing Autopilot profiles that target assigned or dynamic groups with Microsoft Managed Desktop devices. Microsoft Managed Desktop uses Autopilot to provision new devices. If you have an existing autopilot deployment profile, the setting "Convert all targeted devices to Autopilot" must be set to "No" for the managed desktop readiness test for autopilot to succeed.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
You shouldn't have any existing Autopilot profiles that target assigned or dynamic groups with Microsoft Managed Desktop devices. Microsoft Managed Desktop uses Autopilot to provision new devices. If you have an existing Autopilot deployment profile, the "Convert all targeted devices to Autopilot" setting must be set to "No" for the managed desktop readiness test for Autopilot to succeed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks

**Pull request merged**